src/cs/Directory.Build.props

```diff
@@ -3,9 +3,13 @@
 	<Import Project="../../Directory.Build.props" />
 
 	<PropertyGroup>
-		<TargetFrameworks>netstandard2.0;netstandard2.1;net6.0</TargetFrameworks>
+		<TargetFrameworks>netstandard2.1;net6.0</TargetFrameworks>
+		<TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```

Comment:
Does this just replace the value for `TargetFrameworks` on line 6?

Reply:
Oh, that is a copy-paste mistake. Line 6 (with `netstandard2.1`) is the correct one for the library projects, though it doesn't make a big difference since either `netstandard2.1` or `netcoreapp3.1` can run on both .NET Core 3.1 and .NET 5.

I'll make a PR to fix it.

Reply:
https://github.com/microsoft/dev-tunnels-ssh/pull/27